rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverterSpec.groovy

```diff
@@ -99,7 +106,7 @@ class PackageNameConverterSpec extends Specification {
         parsedRpmPackageName)
 
     then:
-      parsedDebPackageName == parsedRpmPackageName
+      parsedDebPackageName != parsedRpmPackageName
```

Comment:
I pulled qualifiedName into a method, so going back to quals which makes more sense anyway.